foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_10_0/SparkPipelineMessagingPomMigration.java

```diff
@@ -66,18 +66,6 @@ protected boolean performMigration(File pomFile) {
         return PomHelper.writeModifications(pomFile, modifications.finalizeMods());
     }
 
-    /**
-     * Checks if a given pom model contains any dependencies with a provided group ID and artifact ID
-     */
-    private boolean hasDependency(Model model, String groupId, String artifactId) {
-        return model.getDependencies()
-             .stream()
-             .anyMatch(dependency ->
-                 dependency.getGroupId().equals(groupId) &&
-                 dependency.getArtifactId().contains(artifactId)
-             );
-    }
-
     private String getDependencyContent(String ignore) {
```

Comment:
Q: should the `ignore` param be removed?

Reply:
The content producer methods for all the pom modification are required to take a string parameter. The parameter was initially meant to be the indent size, but we are setting the indent earlier in the migration and using our own instead. So we can safely ignore the string parameter, but still have to accept it as part of the method.

Reply:
Gotcha. Thanks for the explanation.